### PR TITLE
Create Formatting_Graph_Legends

### PR DESCRIPTION
**Pull request by jholste**
Adding a LOST article for formatting legends for both R and Stata.

### CODE REVIEW

**Comment by NickCH-K** [comment]
This is awesome (I had no idea you could title legends in ggplot labs())! I think two small things would really make it better :

1) how to remove legends in ggplot2
2) a quick application of legend(lab) in stata to a multipart two-way graph combined with || or ()

Could you add these?

File: Presentation/Formatting_Graph_Legends [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+parent: "Presentation"
+mathjax: true
+---
+
+## Formatting Graph Legends
+
+Graph legends are an important part of data representation, as they enable the reader to better under stand the figure infront of them while not cluttering the figure itself. Legends can often be seen next to graphs describing some of aspect of a graph.
+For more information on graphs see other articles such as: bar graphs, scatter plots and histograms.
+
+# GRaphs legends in R
```
Comment (grantmcdermott) [inline Presentation/Formatting_Graph_Legends]: Typo: GRaphs

File: Presentation/Formatting_Graph_Legends [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Here are some of the packages that will be used in this section:
+```{r}
+if (!require("pacman")) install.packages("pacman")
+pacman::p_load(tidyverse, broom, knitr, ggplot2)
```
Comment (grantmcdermott) [inline Presentation/Formatting_Graph_Legends]: @jholste Maybe I'm missing something, but why do you need all these extra packages? Surely you only need ggplot2?

Comment (jholste) [reply Presentation/Formatting_Graph_Legends]: I'll fix that, force of habit

**Comment by grantmcdermott** [comment]
@jholste Have you finished with the minor changes? If so, can you commit them?

**Comment by jholste** [comment]
Hi Grant,
Long time no see, hope you are doing well!
Sorry, with the whole virus thing I have neglected to finish my 
revisions and have not checked git hub for a while. I plan on getting 
those online no later than Sunday morning.
Thank you for the reminder,
James

**Comment by NickCH-K** [comment]
This is great! Merging now.

**Pull request merged**